lib/widgets/icons.dart

```diff
@@ -24,179 +24,182 @@ abstract final class ZulipIcons {
   //
   // BEGIN GENERATED ICON DATA
 
+  /// The Zulip custom icon "archive".
+  static const IconData archive = IconData(0xf101, fontFamily: "Zulip Icons");
```

Comment:
How did you create the link to the design discussion, in the commit message? The channel-ID part is wrong, so it doesn't take me to the intended message.